packages/theming/atlas/src/themesource/atlas_core/web/main.scss

```diff
@@ -1,25 +1,24 @@
 // Main utilities
-@import "mixins/conditional-includes";
+@use "sass:meta";
+@use "mixins/conditional-includes" as *;
 
 // Variables
-@import "variables";
-@import "core-variables";
-@import "../../../theme/web/custom-variables";
+@use "../../../theme/web/exclusion-variables" as *;
 
 // Utilities
-@include include-styles("core/_legacy/bootstrap/bootstrap", $exclude-bootstrap);
-@include include-styles("core/_legacy/bootstrap/bootstrap-rtl", $exclude-bootstrap-rtl);
-@include include-styles("core/_legacy/mxui", $exclude-mxui);
+@include meta.load-css("core/_legacy/bootstrap/bootstrap");
```

Comment:
Isn't the change here _always_ including the scss, counter to the goal of conditional imports? Ditto for similar changes in this file.

Reply:
It seems you have removed "general exclusions"; has this been dropped as a requirement?

Reply:
Yes, we decided that during the alignment meeting. It doesn't make much sense to disable to core of Atlas. Everything including page templates and building blocks are built on top of that. It's better to not include atlas as a module at all, if you want to disable those things.